synthetics_AWS/terraform/README.md

```diff
@@ -0,0 +1,35 @@
+# Synthetics agent deployment on AWS ec2 instance
+
+The module creates ec2 instance with provided configuration and deploys `ksynth` agent.
+
+## Requirements
+
+| Name         | Version  |
+|--------------|----------|
+| terraform    | >= 1.0.0 |
```

Comment:
better peg the major version to avoid incompatible changes when new major version is released